packages/modal/addon/components/nucleus-modal/dialog.js

```diff
@@ -225,9 +225,11 @@ class Dialog extends Component {
     if (currentIndex === -1) {
       focusElements[0].focus();
     }
-    if(event.shiftKey && currentIndex === 0) {
-      event.preventDefault();
-      focusElements[focusElements.length-1].focus();
+    if(event.shiftKey) {
+      if (currentIndex === 0) {
```

Comment:
why nested if?

Reply:
Because we want the last focusable element to fall into the top-level `if(event.shiftKey)` If it is not nested, this condition is evaluated as false for the last focusable element and it will fall into the else if on `line 232` which will make it focus on the first element **even if** Shift key is pressed (which is not the intended result)